pallets/elections/src/impls.rs

```diff
@@ -52,34 +61,34 @@ fn rotate<T: Clone + PartialEq>(
     current_era: EraIndex,
     current_session: SessionIndex,
     n_validators: usize,
-    all_validators: Vec<T>,
     reserved: Vec<T>,
+    non_reserved: Vec<T>,
 ) -> Option<Vec<T>> {
     if current_era == 0 {
```

Comment:
Maybe return when `non_reserved.is_empty()`, then you don't need `current_era` in this method and in `rotate_members`.

Reply:
done